MapboxDirections/MBRouteStep.swift

```diff
@@ -371,6 +371,60 @@ public enum ManeuverDirection: Int, CustomStringConvertible {
     }
 }
 
+extension String {
+    internal func tagValuesSeparatedByString(separator: String) -> [String] {
+        return componentsSeparatedByString(separator).map { $0.stringByTrimmingCharactersInSet(.whitespaceCharacterSet()) }.filter { !$0.isEmpty }
+    }
+}
+
+/**
+ Encapsulates all the information about a road.
+ */
+struct Road {
+    let names: [String]?
+    let codes: [String]?
+    let destinations: [String]?
+    let destinationCodes: [String]?
+    
+    init(name: String?, ref: String?, destination: String?) {
+        var codes: [String]?
+        if let names = name, let gloss = ref ?? destination {
```

Comment:
I don't understand the semantic meaning of `gloss` and the dictionary did not help me understand. The case that in Mapbox Directions v5 `destination` is mixed into `name` can not exist. Likewise it makes no sense to me that `codes` would ever have the contents of `destination` in them.